Models/UIRegionRect.cs

```diff
@@ -35,28 +34,29 @@ public new double Y
             set
             {
                 base.Y = value;
-                OnPropertyChanged(nameof(Y));
                 OnPropertyChanged(nameof(ScaledY));
             }
         }
 
+
         public new double Height
         {
             get => base.Height;
             set
             {
                 base.Height = value;
-                OnPropertyChanged(nameof(Height));
```

Comment:
Why did we remove this one?

Reply:
because OnPropertyChanged is passed up from base.Height and otherwise it is executed twice.